src/helix-ui/styles/modules/choice-tiles.less

```diff
@@ -7,8 +8,20 @@
 // Layout & Geometry
 /* ========== COMMON ========== */
 label.hxChoice > input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
```

Comment:
This property isn't supported across all browsers, so we need to include the browser prefixes, too.